students.log

```diff
@@ -0,0 +1,251 @@
+==119704== Memcheck, a memory error detector
+==119704== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
+==119704== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
+==119704== Command: ./main
+==119704== Parent PID: 38583
+==119704== 
+==119704== Use of uninitialised value of size 8
```

Comment:
Many logs here.
Have you managed to resolve them already?

Reply:
Not yet, 
will try to get to it ASAP.

Reply:
It's ok.
Focus on the current project.
I know how complicated is the Mobileye project, so get back to this when you are done.
Good Luck!